index.js

```diff
@@ -2,7 +2,7 @@
 
 var assign = require('object.assign');
 
-var modules = [require('babel-plugin-transform-es2015-modules-commonjs'), {
+var modules = [require('@babel/plugin-transform-modules-commonjs'), {
   strictMode: true, // add "use strict"
```

Comment:
These two options are already the default behavior, including them shouldn't be needed.

Reply:
Removed them

Reply:
I'd prefer to include them for explicitness.